compiler/z/codegen/OMRInstOpCodeProperties.hpp

```diff
@@ -4886,7 +4886,7 @@
    {
```

Comment:
Could you update the copyright header in this file please?

Reply:
Fixed in [force-push](https://github.com/eclipse/omr/compare/aa4d2cf9e7c421a5ea39e9f8d701fd14e58ddbca..0b3376f0ba6f553545daf6908b91dc0a9729d50e).